INN/umbrella-energynewsnetwork
# Regularize the styles for the MWEN Mailchimp Subscribe widget

## Changes

This pull request makes the following changes:

- Regularizes the styles for the "ENN Mailchimp Signup" widget in the "Header Ad Zone" widget area.

Article:
![Screen Shot 2020-02-18 at 18 46 23](https://user-images.githubusercontent.com/1754187/74789155-de2d7000-5281-11ea-946a-8b35af95f2b6.png)

Term archive:
![Screen Shot 2020-02-18 at 18 46 33](https://user-images.githubusercontent.com/1754187/74789156-dec60680-5281-11ea-8bcd-45a6581a9cef.png)

Homepage:
![Screen Shot 2020-02-18 at 18 46 12](https://user-images.githubusercontent.com/1754187/74789153-dd94d980-5281-11ea-88ea-c8a55a288bed.png)

## Why



For #84

## Testing/Questions

Features that this PR affects:

- Homepage, article pages, term pages


Questions that need to be answered before merging:


Steps to test this PR:

1. In Appearance > Theme Options > Advanced, check "Enable Optional Leaderboard Ad Zone"
2. To the Header Ad Zone widget area, add one ENN Mailchimp Signup Widget, and set it to hide on desktops and tablets.
